src/java_binding/java/pom.xml

```diff
@@ -40,12 +40,12 @@
                 <version>${grpc.version}</version>
             </dependency>
             <dependency>
-                <groupId>com.risingwave.java</groupId>
+                <groupId>${project.groupId}</groupId>
```

Comment:
It's not a common practice to set the group id as a property. Usually we use property as a place holder for things that are used in many places but likely to change in the future, such as the version id. The group id is not likely to change, so we'd better use its own name.